resolwe_bio/tests/processes/test_support_processors.py

```diff
@@ -548,3 +548,33 @@ def test_umi_tools_dedup(self):
 
         dedup_paired = self.run_process('umi-tools-dedup', {'alignment': bam_paired.id})
         self.assertFile(dedup_paired, 'stats', './corall/output/dedup_paired_stats.txt')
+
+    @tag_process('seqtk-rev-complement-single', 'seqtk-rev-complement-paired')
+    def test_reverse_complement(self):
+        with self.preparation_stage():
+            reads = self.run_process('upload-fastq-single', {
```

Comment:
You can use `self.prepare_reads` helper function which will match your usage of `self.prepare_paired_reads`.